### PR TITLE
add truststore dependency

### LINKED ISSUE #726: gcalcli should support locally-installed certificates (via truststore)
Currently gcalcli won't work on machines that have local certificates installed, e.g. in corporate environments where a firewall requires certificates installed on internal boxes. Hooking up [truststore](https://pypi.org/project/truststore/) or something similar would fix that.

### PR DESCRIPTION
**Pull request by ajkessel**
gcalcli won't work on machines that have local certificates installed, e.g. in corporate environments where a firewall requires certificates installed on internal boxes. This is a simple fix that tells gcalcli to trust the local cert store. Perhaps this should be made an optional feature or there is some better/more secure way to do it, but I would very much appreciate a way to trust the local certificate store in the upstream code.

Fixes #726.

### CODE REVIEW

File: gcalcli/cli.py [hunk 1]
```diff
@@ -20,6 +20,8 @@
 # Everything you need to know (Google API Calendar v3): http://goo.gl/HfTGQ #
 #                                                                           #
 # ######################################################################### #
+import truststore
+truststore.inject_into_ssl()
```
Comment (dbarnett) [inline gcalcli/cli.py:24]: Linter doesn't like this. I'd move the `inject_into_ssl()` call down into main() and add a small comment about what it does, then move the import down to be sorted.

File: pyproject.toml [hunk 2]
```diff
@@ -26,6 +26,7 @@ classifiers = [
   "Programming Language :: Python :: 3",
 ]
 dependencies = [
+  "truststore",
```
Comment (dbarnett) [inline pyproject.toml:29]: Nit: Could you sort these dependencies above the "dev dependencies" comment?

**Review by dbarnett — CHANGES_REQUESTED** [review]
Using truststore looks fine to me, just needs a few cleanups and it should be fine to merge.

**Comment by dbarnett** [comment]
Also might be nice to have an opt-out for this if there's any chance it could behave badly, but I think it's okay to add that later as needed.

**Comment by ajkessel** [comment]
I'm fairly new to this so not sure if I did it correctly, but I attempted to address your feedback.

**Review by dbarnett — APPROVED** [review]
Looks great! Merging...

**Comment by dbarnett** [comment]
Err, looks like we'll need to either bump the minimum required python version or make this dep optional. I was thinking to bump it to 3.10 anyway. Want me to do that separately and then you can rebase your changes onto that?

**Comment by ajkessel** [comment]
> Err, looks like we'll need to either bump the minimum required python version or make this dep optional. I was thinking to bump it to 3.10 anyway. Want me to do that separately and then you can rebase your changes onto that?

Sure, that works!

**Comment by dbarnett** [comment]
> I was thinking to bump it to 3.10 anyway.

Done: #727.

**Pull request closed without merging**